test/unittests/cxx20_bit_test.cpp

```diff
@@ -47,20 +48,12 @@ TEST(cxx20_bit, bit_cast_uint32_to_array)
 
 TEST(cxx20_bit, popcount32)
 {
-    EXPECT_EQ(popcount(uint32_t{0}), 0);
-    EXPECT_EQ(popcount(uint32_t{0xffffffff}), 32);
-    EXPECT_EQ(popcount(uint32_t{0xffff0000}), 16);
-    EXPECT_EQ(popcount(uint32_t{0x0000ffff}), 16);
-    EXPECT_EQ(popcount(uint32_t{0x00ffff00}), 16);
-    EXPECT_EQ(popcount(uint32_t{0x00ff00ff}), 16);
+    for (const auto& [input, expected] : test::popcount32_test_cases)
+        EXPECT_EQ(popcount(input), expected);
 }
 
 TEST(cxx20_bit, popcount64)
 {
-    EXPECT_EQ(popcount(uint64_t{0}), 0);
-    EXPECT_EQ(popcount(uint64_t{0xffffffffffffffff}), 64);
-    EXPECT_EQ(popcount(uint64_t{0xffffffff00000000}), 32);
-    EXPECT_EQ(popcount(uint64_t{0x00000000ffffffff}), 32);
-    EXPECT_EQ(popcount(uint64_t{0x0000ffffffff0000}), 32);
-    EXPECT_EQ(popcount(uint64_t{0x00ff00ff00ff00ff}), 32);
+    for (const auto& [input, expected] : test::popcount64_test_cases)
```

Comment:
Should we have `using namespace test;` at the top instead?

Reply:
Can be. Up to you.

Reply:
Did that then.